modules/@apostrophecms/i18n/ui/apos/apps/AposI18nCrossDomainSession.js

```diff
@@ -0,0 +1,23 @@
+export default () => {
```

Comment:
This looks like normal `ui/src/*.js` client side JS, but it's in an `ui/apos/apps` directory. I'm guessing that's to keep it in the apos scene. Don't we have `ui/apos/js` or something like that?

Reply:
Apostrophe loads admin UI "apps" from `ui/apos/apps`, as opposed to Vue components which are loaded and registered by name from `ui/apos/components`. Apps are automatically invoked, while component .vue files are just components for use by apps and other components.

So admin UI "apps" work similarly to `ui/src/index.js` files but are not loaded for logged-out users. Not a new thing in this PR.

Reply:
I realize it's not a new thing. It seems like we might have had a blind spot, however. This isn't an app like the Vue apps are, but the file path suggests that it would be. If we don't have a place for logged-in client JS that's not instantiating a Vue app that seems to be a gap.